tests/Mono.Android-Tests/Xamarin.Android.Net/AndroidMessageHandlerTests.cs

```diff
@@ -61,9 +61,11 @@ public async Task ServerCertificateCustomValidationCallback_RejectRequest ()
 
 			try {
 				await client.GetStringAsync ("https://microsoft.com/");
-			} catch {
+			} catch (System.Net.WebException) {
 				// System.Net.WebException is thrown in Debug mode
-				// Java.Security.Cert.CertificateException is thrown in Release mode
+				exceptionWasThrown = true;
+			} catch (Java.IO.IOException) {
+				// Java.IO.IOException is thrown in Release mode
```

Comment:
Wait, what?  Why is `Java.IO.IOException` thrown in Release mode?  (Callstack?)  I would hope that Debug & Release configs have consistent behavior when possible, and this comment calls that into question.

Reply:
The Java exception is mapped into WebException when `JNIEnv.ShouldWrapJavaException (ex)` is true. Now that I looked into it again, I realize that I probably mistook Debug vs. Release with legacy vs. .net and the comments are incorrect. I'll update the comments to make sense.